src/components/charts/SummaryPieChart.jsx

```diff
@@ -0,0 +1,61 @@
+
+import {
+  ResponsiveContainer, PieChart, Pie, Legend, Tooltip, Cell,
+} from 'recharts';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localTheme } from '../../localconf';
```

Comment:
again, the component should be self-contained don't use localconf or localtheme. It should not aware of any configuration of the project